test/controllers/urls_controller_test.rb

```diff
@@ -2,7 +2,7 @@
 
 require "test_helper"
 
-class UrlControllerTest < ActionDispatch::IntegrationTest
+class UrlsControllerTest < ActionDispatch::IntegrationTest
```

Comment:
I haven't changed name of any controller test. Practically, they are testing PushController, not related controller. Their name can be changed with next PRs, not with this PR.